pom.xml

```diff
@@ -80,7 +80,7 @@
     <google.cloud.bigtable.version>1.17.1</google.cloud.bigtable.version>
     <google.cloud.bigquery.version>1.137.1</google.cloud.bigquery.version>
     <google.cloud.kms.version>2.0.2</google.cloud.kms.version>
-    <google.cloud.pubsub.version>1.108.1</google.cloud.pubsub.version>
+    <google.cloud.pubsub.version>1.112.1</google.cloud.pubsub.version>
```

Comment:
Why are we updating the pubsub and avro version in dev dependency ?

Reply:
We need this is the old version and it is not compatible with various components in the code.